caluma/form/validators.py

```diff
@@ -70,6 +71,27 @@ def _validate_question_multiple_choice(self, question, value, **kwargs):
                 f"Should be one of the options [{', '.join(options)}]"
             )
 
+    def _validate_question_dynamic_choice(self, question, value, **kwargs):
+        options = get_data_source_data(document, question.data_source).values_list(
```

Comment:
Are we sure we want to do this? The idea of dynamic options is, well, that they are dynamic :)
So a valid option that existed at the time of loading the question could be an invalid one at the time of validation.

I would reduce this validation to just:

```python
if not isinstance(value, str):
    raise Exception()
```

Same goes for `_validate_question_dynamic_multiple_choice`.

Reply:
Hmm, good point. Our current use case would prefer this to be valid though, the list is not expected to change frequently.

Reply:
So we expect this to fail just unfrequently ;)

I think we need to come up with a good solution for this.

Personally I'd prefer a lax validation over an error-prone.

Reply:
I see your point. In our specific use-case, validation is also crucial though, because decides the next assigned group in the workflow. Idea: Shall we make this configurable in the data-source? Something like `enforce_validation: True/False`?